xml/FrameworksIndex/dotnet-plat-ext-5.0.xml

```diff
@@ -8591,6 +8591,7 @@
       <Member Id="P:System.Diagnostics.PresentationTraceSources.ResourceDictionarySource" />
       <Member Id="P:System.Diagnostics.PresentationTraceSources.RoutedEventSource" />
       <Member Id="P:System.Diagnostics.PresentationTraceSources.ShellSource" />
+      <Member Id="P:System.Diagnostics.PresentationTraceSources.TraceLevel" />
```

Comment:
these new properties were missing attached properties in docs. You'll need to migrate the existing content to these and ideally delete or turn off indexing to the ones migrated to previous-versions because of this bug.
e.g. https://docs.microsoft.com/en-us/previous-versions/bb778381(v%3dvs.110)

Reply:
Tag @joelmartinez